### PR TITLE
Add experimental notebook consisting of EDA and implemented ML approaches

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
None

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements
EDA and baseline reviewer recommender models done on PullRequest data aggregated from `openshift/origin` repository.

### CODE REVIEW

**Comment by xtuchyna** [comment]
This is related to #2

File: Pipfile [hunk 1]
```diff
@@ -8,6 +8,12 @@ nltk = "*"
 gensim = "*"
 srcopsmetrics = "*"
 pygithub = "*"
+voluptuous = "*"
+jupyter = "*"
```
Comment (pacospace) [inline Pipfile:12]: why jupyter? :) this notebook should be created from Operate First right? :)

Comment (xtuchyna) [reply Pipfile:12]: @pacospace well it's here in case anyone else would want to launch it somewhere else (otherwise it would not have to be on GH right?)

**Comment by pacospace** [comment]
can you please add title, description of the notebook, maybe sections to differentiate what you are doing with more descriptions in the notebook?

File: Pipfile (hunk 1 above)
Comment (pacospace) [reply Pipfile:12]: You can describe that in the README: install jupyter or jupyterlab ot go to Operate First and spawn this image, etc etc, but I feel it should not be part of that.

**Comment by pacospace** [comment]
> /cc @pacospace

https://github.com/thoth-station/socrates/pull/10/files#r691885062

**Comment by pacospace** [comment]
> /cc @pacospace is it okay now?

Conclusion in the notebook are missing :)

**Comment by pacospace** [comment]
> > > /cc @pacospace is it okay now?
> > 
> > 
> > Conclusion in the notebook are missing :)
> 
> Added, check it out :)

Few more comments :) 
- The notebook has a broader focus rather than EDA, is that expected? or it should be two different notebooks?
- Usually imports should go at the top in the first notebook cells
- The initial description of the notebook can be extended to be more verbose, wdyt?

**Review by pacospace — APPROVED** [review]
lgtm!

**Comment by harshad16** [comment]
@xtuchyna Is this relevant
can you once rebase and push 
thank you

**Pull request closed without merging**